medusa.py

```diff
@@ -1104,6 +1104,9 @@ def do_run(self, line) -> None:
                     pattern = r'\b\d+\b'
                     get_pid = re.findall(pattern, option)
                     self.run_frida(False, False, '', self.device, get_pid[0], host, port)
+                elif flags[0] == '-F':
```

Comment:
Is it ok to change the flag to '-t' , trying to keep somekind of a convention

Reply:
-F is trying to keep consistent with flags from frida-python,
-f in this function is already 'Initiate a Frida session and spawn the selected package', which is "spawn"; while attaching to topmost application is "attach"

Reply:
'-t' its more like an abbreviation for "top", so it is easier to remember. I am not keeping the frida flags (at least not trying to).

Reply:
That would be a nice addition.